include/formbase.php

```diff
@@ -201,9 +201,23 @@ function getPostToForm($ignore='', $isRegularExpression=false)
 
 function getGetToForm($ignore='', $usePostAsAuthority = false)
 {
+    global $log;
 	$fields = '';
-	foreach ($_GET as $key=>$value)
+	foreach ($_GET as $key => $value)
 	{
+	    if(is_array($key)) {
+            if(!empty($key)) {
```

Comment:
You don't need to use !empty if you just want to warn the user $key must be a string.

Reply:
This code is to remove the PHP notice array conversion to string. There is an instance where an empty array is being passed into the $value. If the array is empty, it doesn't really break anything per se, so we can just ignore it. However if a developer is deliberate passing in an array with values in it. We should inform them that this behavior is incorrect.  Then they can decide what are the next steps they need to take. Since this is code needs to be backwards compatible, we cannot use exceptions. So the alternative is to  write it into the suitecrm.log.